src/pokenav_unk_2.c

```diff
@@ -249,25 +309,967 @@ const struct SpriteTemplate gUnknown_0862036C =
+}
+
+static void sub_81CA698(void)
+{
+    struct Pokenav2Struct * ptr = GetSubstructPtr(2);
+    DestroySprite(ptr->field_028);
+}
+
+static void sub_81CA6AC(struct Sprite * sprite)
+{
+    sprite->data[0]++;
+    if (sprite->data[0] > 8)
+    {
+        sprite->data[0] = 0;
+        sprite->invisible ^= 1;
+    }
+}
+
+static void sub_81CA6E0(void)
+{
+    struct Pokenav2Struct * ptr = GetSubstructPtr(2);
+
+    ptr->field_008 = AddWindow(&gUnknown_086202CC);
+    PutWindowTilemap(ptr->field_008);
+    FillWindowPixelBuffer(ptr->field_008, PIXEL_FILL(6));
+    CopyWindowToVram(ptr->field_008, 3);
+}
+
+static void sub_81CA714(void)
+{
+    struct Pokenav2Struct * ptr = GetSubstructPtr(2);
+    int i = sub_81C98B4();
+    const u8 * s = gUnknown_086202D4[i];
+    u32 width = GetStringWidth(1, s, -1);
+    FillWindowPixelBuffer(ptr->field_008, PIXEL_FILL(6));
+    AddTextPrinterParameterized3(ptr->field_008, 1, (192 - width) / 2, 1, gUnknown_0862030C, 0, s);
+}
+
+
+static void sub_81CA770(void)
+{
+    struct Pokenav2Struct * ptr = GetSubstructPtr(2);
+    const u8 * s = gText_NoRibbonWinners;
+    u32 width = GetStringWidth(1, s, -1);
+    FillWindowPixelBuffer(ptr->field_008, PIXEL_FILL(6));
+    AddTextPrinterParameterized3(ptr->field_008, 1, (192 - width) / 2, 1, gUnknown_0862030F, 0, s);
+}
+
+static bool32 sub_81CA7C4(void)
+{
+    return IsDma3ManagerBusyWithBgCopy();
+}
+
+static void sub_81CA7D4(void)
+{
+    struct Pokenav2Struct * ptr = GetSubstructPtr(2);
+    ptr->field_00a = CreateTask(sub_81CA808, 2);
+}
+
+static void sub_81CA7F4(void)
+{
+    struct Pokenav2Struct * ptr = GetSubstructPtr(2);
+    DestroyTask(ptr->field_00a);
+}
+
+static void sub_81CA808(u8 taskId)
+{
+    ChangeBgX(3, 0x80, 1);
+}
+
+static void sub_81CA818(void)
+{
+    u8 taskId = CreateTask(sub_81CA8B0, 3);
+    SetWordTaskArg(taskId, 1, (uintptr_t)(gUnknown_0861FC78 + 1));
+    SetWordTaskArg(taskId, 3, (uintptr_t)(gUnknown_0861FC78 + 7));
+}
+
+static void sub_81CA850(void)
+{
+    CopyPaletteIntoBufferUnfaded(gUnknown_0861FC78 + 7, 0x31, 4);
+}
+
+static void sub_81CA864(void)
+{
+    u8 taskId = CreateTask(sub_81CA8B0, 3);
+    SetWordTaskArg(taskId, 1, (uintptr_t)(gUnknown_0861FC78 + 7));
+    SetWordTaskArg(taskId, 3, (uintptr_t)(gUnknown_0861FC78 + 1));
+}
+
+static bool32 sub_81CA89C(void)
+{
+    return FuncIsActiveTask(sub_81CA8B0);
+}
+
+static void sub_81CA8B0(u8 taskId)
+{
+    u16 sp8[2];
+    s16 * data = gTasks[taskId].data;
+    const u16 * pal1 = (const u16 *)GetWordTaskArg(taskId, 1);
+    const u16 * pal2 = (const u16 *)GetWordTaskArg(taskId, 3);
+
+    sub_81C79BC(pal1, pal2, 2, 12, ++data[0], sp8);
+    LoadPalette(sp8, 0x31, 4);
+    if (data[0] == 12)
+        DestroyTask(taskId);
+}
+
+static void sub_81CA914(void)
+{
+    TransferPlttBuffer();
+    LoadOam();
+    ProcessSpriteCopyRequests();
+    ScanlineEffect_InitHBlankDmaTransfer();
+}
+
+static void titlescreen_0(void) // almost definitely the wrong name
```

Comment:
Could you just change it to sub_8XXXXXX instead, this is definitely a wrong name lol. 
Btw I remember I had a function called unused_1 in dodrio berry picking.